include/sys/zvol_impl.h

```diff
@@ -60,6 +60,30 @@ typedef struct zvol_state {
 	boolean_t		zv_threading;	/* volthreading property */
 } zvol_state_t;
 
+/*
+ * zvol taskqs
+ */
+typedef struct zv_taskq {
+	uint_t tqs_cnt;
+	taskq_t **tqs_taskq;
+} zv_taskq_t;
+
+typedef struct zv_request_stack {
+	zvol_state_t	*zv;
+	struct bio	*bio;
+	struct request	*rq;
```

Comment:
`struct request` is a Linux thing.  And I suppose it is only a coincidence that both FreeBSD and Linux both have `struct bio`.

Reply:
Place `struct request *rq` declaration under Linux ifdef.